manager/gui/src/components/configuration/LrSystemHealthConfiguration.vue

```diff
@@ -0,0 +1,368 @@
+                ></v-checkbox>
+              </v-col>
+            </v-row>
+
+            <v-divider></v-divider>
+            <v-row class="field-row">
+              <v-col cols="3"></v-col>
+              <v-col cols="4" class="button-col">
+                <v-btn
+                  class="mr-5"
+                  color="primary"
+                  @click="applyNotificationSettings"
+                  :disabled="disableApply"
+                  :dark="isSettingsValid && isVerified"
+                  data-test="apply-btn"
+                  >Apply
+                </v-btn>
+                <v-btn
+                  color="csmdisabled"
+                  @click="resetConfirmation()"
+                  depressed
+                  dark
+                  data-test="reset-btn"
+                  >Reset</v-btn
+                >
+              </v-col>
+            </v-row>
+          </v-form>
+        </v-expansion-panel-content>
+      </v-expansion-panel>
+    </v-expansion-panels>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+import SgtDropdown from "@/lib/components/SgtDropdown/SgtDropdown.vue";
+import SgtDropFile from "@/lib/components/SgtDropFile/SgtDropFile.vue";
+import { Api } from "@/services/Api";
+import SgtDialog from "@/lib/components/SgtDialog/SgtDialog.vue";
+import { SgtDialogModel } from "@/lib/components/SgtDialog/SgtDialog.model";
+import { create } from "vue-modal-dialogs";
+import SgtTooltipIcon from "@/lib/components/SgtTooltipIcon/SgtTooltipIcon.vue";
+import { emailProtocols } from "@/utils/CommonUtil.constant";
+import {
+  emailRegex,
+  passwordRegex,
+  ipAddressRegex,
+  fqdnRegex,
+} from "@/utils/RegexHelpers";
+
+@Component({
+  name: "LrSystemHealthConfiguration",
+  components: { SgtDropdown, SgtDropFile, SgtTooltipIcon },
+})
+export default class LrSystemHealthConfiguration extends Vue {
+  panel = 0;
+  test = "";
+  protocolOptions: string[] = [];
+  resetModal = create<SgtDialogModel>(SgtDialog);
+  isVerified = false;
+  isSettingsValid = false;
+
+  notificationSettings = {
+    server: null,
+    port: null,
+    protocol: null,
+    senderEmail: null,
+    senderPassword: null,
+    confirmPassword: null,
+    receiverEmails: null,
+  };
+
+  validationRules = {
+    serverAddress: [
+      (val: string) => (val || "").length > 0 || "This field is required",
+      (val: string) =>
+        ipAddressRegex.test(val) || fqdnRegex.test(val) || "Invalid value",
+    ],
+    portNumber: [
+      (val: string) => (val || "").length > 0 || "This field is required",
+      (val: string) => +val <= 65536 || "Invalid value",
+    ],
+    email: [
+      (val: string) => (val || "").length > 0 || "Email is required",
+      (val: string) => emailRegex.test(val) || "Invalid email",
+    ],
+    password: [
+      (val: string) => (val || "").length > 0 || "Password is required",
+      (val: string) => passwordRegex.test(val) || "Invalid password",
```

Comment:
need a validation to compare the password to confirm password here also if confirm password exist.